webdriver-spec.html

```diff
@@ -4624,636 +4646,562 @@
-      <tr><td>NUMPAD1</td><td>\uE01B</td><td>Special key</td></tr>
-      <tr><td>NUMPAD2</td><td>\uE01C</td><td>Special key</td></tr>
-      <tr><td>NUMPAD3</td><td>\uE01D</td><td>Special key</td></tr>
-      <tr><td>NUMPAD4</td><td>\uE01E</td><td>Special key</td></tr>
-      <tr><td>NUMPAD5</td><td>\uE01F</td><td>Special key</td></tr>
-      <tr><td>NUMPAD6</td><td>\uE020</td><td>Special key</td></tr>
-      <tr><td>NUMPAD7</td><td>\uE021</td><td>Special key</td></tr>
-      <tr><td>NUMPAD8</td><td>\uE022</td><td>Special key</td></tr>
-      <tr><td>NUMPAD9</td><td>\uE023</td><td>Special key</td></tr>
-      <tr><td>MULTIPLY</td><td>\uE024</td><td>Special key</td></tr>
-      <tr><td>ADD</td><td>\uE025</td><td>Special key</td></tr>
-      <tr><td>SEPARATOR</td><td>\uE026</td><td>Special key</td></tr>
-      <tr><td>SUBTRACT</td><td>\uE027</td><td>Special key</td></tr>
-      <tr><td>DECIMAL</td><td>\uE028</td><td>Special key</td></tr>
-      <tr><td>DIVIDE</td><td>\uE029</td><td>Special key</td></tr>
-      <tr><td>F1</td><td>\uE031</td><td>Special key</td></tr>
-      <tr><td>F2</td><td>\uE032</td><td>Special key</td></tr>
-      <tr><td>F3</td><td>\uE033</td><td>Special key</td></tr>
-      <tr><td>F4</td><td>\uE034</td><td>Special key</td></tr>
-      <tr><td>F5</td><td>\uE035</td><td>Special key</td></tr>
-      <tr><td>F6</td><td>\uE036</td><td>Special key</td></tr>
-      <tr><td>F7</td><td>\uE037</td><td>Special key</td></tr>
-      <tr><td>F8</td><td>\uE038</td><td>Special key</td></tr>
-      <tr><td>F9</td><td>\uE039</td><td>Special key</td></tr>
-      <tr><td>F10</td><td>\uE03A</td><td>Special key</td></tr>
-      <tr><td>F11</td><td>\uE03B</td><td>Special key</td></tr>
-      <tr><td>F12</td><td>\uE03C</td><td>Special key</td></tr>
-      <tr><td>META</td><td>\uE03D</td><td>Special key</td></tr>
-      <tr><td>COMMAND</td><td>\uE03D</td><td>Special key</td></tr>
-      <tr><td>ZENKAKU_HANKAKU</td><td>\uE040</td><td>Special key</td></tr>
-    </table>
-    <p>The keys considered upper-case symbols are either defined by the current keyboard locale or are derived from the US 104 keys Microsoft Windows keyboard layout, which are:
-    <ul>
-      <li>A - Z</li>
-      <li>!$^*()+{}:?|~@#%_\&quot; &amp; &lt; &gt;</li>
-    </ul>
-    <p>When the user input is emulated natively (see note below), the implementation SHOULD use the current keyboard locale to determine which symbols are upper case. In all other cases, the implementation MUST use the US 104 key Microsoft Windows keyboard layout to determine those symbols.</p>
-    <p>The state of the physical keyboard MUST NOT affect emulated user input.</p>
-    <h3>Internationalized input</h3>
-    <p>Non-latin symbols: TBD</p>
-    <p>Complex scripts using Input Method Editor (IME): TBD</p>
-  </section> <!-- sendkeys end -->
- </section> <!-- /High-Level APIs -->
-</section> <!-- /Interactions -->
+
+
+    <p>The steps for <dfn>processing keystrokes</dfn> are:
```

Comment:
I have corrected this and will be also used when `sendKeysToElement` is finished off.